src/kOS/Binding/FlightControlManager.cs

```diff
@@ -156,6 +160,88 @@ private bool VesselIsValid(Vessel vessel)
             return vessel != null && vessel.rootPart != null;
         }
 
+        public void SelectAutopilotMode(object autopilotMode)
+        {
+            if (autopilotMode is Direction)
+            {
+                //TODO: implment use of direction subclasses.
```

Comment:
It is where we will implement a parallel route to the current' steering.  @Dunbaratu and I seem to agree that we don't want to choose an exclusive control option.